template/src/[namespace]-[name].js

```diff
@@ -51,7 +55,7 @@ class [Namespace][Name] extends LitElement {
   // function that renders the HTML and CSS into  the scope of the component
   render() {
     return html`
-      <div class=${this.cssClass}>
+      <div class=${this.cssClass} tabindex="0">
```

Comment:
Why is this here? We shouldn't be making a non-interactive div a tab stop.

Reply:
Ah, I see it's to demonstrate the `:host(.focus-visible)` selector. I still don't think it should be in the default template, since it will probably get left there if the dev is inexperienced with a11y.

Reply:
Why would it get left there? It's the demo? You delete all of it. Would you be ok with putting comments in there to DELETE ALL THE DEMO CODE!